.github/workflows/build.yml

```diff
@@ -21,3 +21,5 @@ jobs:
       run: cargo build --verbose --examples
     - name: Run tests
       run: cargo test --verbose
+      env:
+        RUST_BACK_TRACE: 1
```

Comment:
Is that the same as `RUST_BACKTRACE=full`? I see the CI logs suggesting that and noticed that there is a difference in one underscore and in `1` versus `full`. Ignore me if you're already aware.

Reply:
yeah - no that was wrong and was a typo =(. 

I've managed to replicate the OSX failure locally - and I've fixed in https://github.com/benfred/read-process-memory/pull/5/commits/035818ef2b01fd573ac0e678e946672a72d24700  . Apparently I also need to run it as sudo too - but I *think* I've almost got this going =)

Reply:
Cool!